testsuite/shared/src/main/java/org/jboss/as/test/integration/management/cli/AcmeMockServerBuilder.java

```diff
@@ -0,0 +1,560 @@
+        final String FINALIZE_URL = "/acme/finalize/398/186";
+
+        final String FINALIZE_RESPONSE_BODY = "{" + System.lineSeparator() +
+                "  \"status\": \"valid\"," + System.lineSeparator() +
+                "  \"expires\": \"2018-05-04T18:27:35Z\"," + System.lineSeparator() +
+                "  \"identifiers\": [" + System.lineSeparator() +
+                "    {" + System.lineSeparator() +
+                "      \"type\": \"dns\"," + System.lineSeparator() +
+                "      \"value\": \"inlneseppwkfwew.com\"" + System.lineSeparator() +
+                "    }" + System.lineSeparator() +
+                "  ]," + System.lineSeparator() +
+                "  \"authorizations\": [" + System.lineSeparator() +
+                "    \"http://localhost:4001/acme/authz/LJRH3-gjUPt5U5v8wH1Ch3eFcxu8UK-uotjutm5NB9s\"" + System.lineSeparator() +
+                "  ]," + System.lineSeparator() +
+                "  \"finalize\": \"http://localhost:4001/acme/finalize/398/186\"," + System.lineSeparator() +
+                "  \"certificate\": \"http://localhost:4001/acme/cert/ffba1352e17b57c2032136e6729b0c2ebac9\"" + System.lineSeparator() +
+                "}" + System.lineSeparator();
+
+        final String FINALIZE_REPLAY_NONCE = "CZleS8d9p38tiIdjbzLa1PRJIEFcbLevx_jtlZQzYbo";
+        final String FINALIZE_LOCATION = "http://localhost:4001/acme/order/398/186";
+
+        final String CHECK_ORDER_URL = "/acme/order/398/186";
+
+        final String CHECK_ORDER_RESPONSE_BODY = "{" + System.lineSeparator() +
+                "  \"status\": \"valid\"," + System.lineSeparator() +
+                "  \"expires\": \"2018-05-04T18:27:35Z\"," + System.lineSeparator() +
```

Comment:
It is valid to use already expired response?
Does it check the time of expiring?

If yes, it doesn't good to use current time + one day to expire time?

Reply:
The "expires" field here is used by the ACME server (i.e., certificate authority) only. It is a timestamp after which the ACME server will consider the certificate order to be invalid. The mock ACME server that is being used in the tests makes use of pre-recorded messages which is why it doesn't fail even though the date is in the past.